net-libs/libtorrent/files/libtorrent-0.14.0-tests-address.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/test/Makefile.am b/test/Makefile.am
+index 0221796..6e23502 100644
+--- a/test/Makefile.am
++++ b/test/Makefile.am
+@@ -39,8 +39,6 @@ LibTorrent_Test_Common = \
+ 	helpers/utils.h
+ 
+ LibTorrent_Test_Torrent_Net_SOURCES = $(LibTorrent_Test_Common) \
+-	torrent/net/test_address_info.cc \
+-	torrent/net/test_address_info.h \
```

Comment:
Is there a bug somewhere that explains why you want to remove this?

Reply:
This [line](https://github.com/rakshasa/libtorrent/blob/v0.14.0/test/torrent/net/test_address_info.cc#L25) causes a failure :  
```
torrent/net::test_address_info:
torrent/net::test_address_info::test_basic
ai_ref_err got wrong error, expected 'Name or service not known', got 'Temporary failure in name resolution'
 : assertion

[...]

- Expression: test_valid_ai_ref_err(std::bind(torrent::ai_get_addrinfo, "1.1.1.
300", nullptr, nullptr, std::placeholders::_1), EAI_NONAME)

Failures !!!
```

This could be resolved, but I think these tests are beyond the scope of the package.

Reply:
sounds like the network sandbox is blocking it. i agree that i'd rather drop skip the tests than enable the network tho yeah.